com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Please reformat your file as described in https://github.com/flathub/flathub/wiki/YAML-Style-Guide.

Reply:
I wasn't aware this existed, will do!